tests/testerina-unit-test/src/test/resources/logging.properties

```diff
@@ -0,0 +1,37 @@
+#
+# Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# WSO2 Inc. licenses this file to you under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
```

Comment:
Do we need this file?

Reply:
Yes, better to have it since we can see the test execution logs in the console